src/Tizen.Messaging/doc/api/Tizen.Messaging.Email.md

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+[!NOTE]
+All Email APIs have been deprecated since Tizen 8.0 and will be removed after two releases without any alternatives.
+
 uid: Tizen.Messaging.Email
 summary: The Tizen.Messaging.Email namespace contains classes providing the functionality to send emails.
```

Comment:
The Obsolete message says "might be removed" and the note in module documentation says "will be removed". Also, please put the note in the remarks and the note lines in old docfx we are using have to start with `>`.
```suggestion
uid: Tizen.Messaging.Email
summary: The Tizen.Messaging.Email namespace contains classes providing the functionality to send emails.
remarks: |
  > [!NOTE]
  > All Email APIs have been deprecated since Tizen 8.0 (API Level 11) which means they are planned to be removed after two releases.
```